packages/server/src/threads/automation.ts

```diff
@@ -557,6 +572,7 @@ class Orchestrator {
         ;(outputs as any).status = AutomationStatus.STOPPED
       }
 
+      span.addTags({ outputsKeys: Object.keys(outputs) })
```

Comment:
Didn't want to fully add the inputs and outputs as they could be PII, so just adding the keys so we have an idea of the shape of things.